File: android/lib/src/main/java/duck/cameras/android/model/LoginToken.java
package duck.cameras.android.model;

public class LoginToken {
    private final String value;

    public LoginToken(String value) {
        this.value = value;
    }

    public String value() {
        return value;
    }

    public boolean isEmpty() {
        return value == null || value.isEmpty();
    }
}

File: windows/Model/LoginToken.cs
namespace Duck.Cameras.Windows.Model
{
    public class LoginToken
    {
        public LoginToken(string value)
        {
            Value = value;
        }

        public string Value { get; private set; }

        public bool IsEmpty()
        {
            return string.IsNullOrEmpty(Value);
        }
    }
}
